dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/VariableLimiter.mo

```diff
@@ -0,0 +1,103 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain suite of simulation tools for power systems.
+*/
+
+model VariableLimiter"Limit the range of a signal with variable limits"
```

Comment:
Add a space here between model name and comment for readability

Reply:
done